app/uk/gov/hmrc/merchandiseinbaggage/views/EoriNumberView.scala.html

```diff
@@ -27,11 +28,12 @@
 button: components.button
 )
 
-@(form: Form[_], isAgent: YesNo, backButtonUrl: Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(form: Form[_], isAgent: YesNo, backButtonUrl: Call, declarationType: DeclarationType)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
 
 @agentOrTrader = @{if(YesNo.to(isAgent)) "agent" else "trader"}
+@exportOrImport = @{if(declarationType == Export) Some(messages(s"eoriNumber.$agentOrTrader.export.title")) else Some(messages(s"eoriNumber.$agentOrTrader.title")) }
```

Comment:
for imports, we can have key like `imports.title` instead of shorter `.title` then the above can be  written in a single like `eoriNumber.$agentOrTrader.$declarationType.title`

Reply:
See this for example: https://github.com/hmrc/merchandise-in-baggage-frontend/pull/142/files

Reply:
similarly we can have `UserType` trait  with subtypes Agent/Trader .. and then message keys could be `eoriNumber.$userType` but this is something for future, no need to do it in this PR